### ISSUE TITLE
Clicking 'hide axis' twice does not restore axis

### ISSUE THREAD

**Issue by gtfierro** [BTrDB/mr-plotter#3]
Clicking once yields the following in the console:

```
Uncaught TypeError: Cannot read property 'style' of null
applyDisplayColor   @   plot.js:1338
drawYAxes   @   plot.js:975
applySettings   @   plot.js:656
div.append.attr.attr.node.onchange      @   axis.js:161
```

Clicking twice does not bring it back. You need to hit the arrows on either side in order to restore the axis.

**Comment by immesys** [comment]
This is intended behavior,  but the error should be fixed

**Comment by gtfierro** [comment]
OK, that makes sense. I think it would be more consistent to remove the 'hover' on it when it is selected though.

**Comment by samkumar** [comment]
Thanks for finding this. Fixed as of f6b694387d004352015556422d6f5d641823ea5f.

Clicking the 'hide axis' when it is already selected is not intended to do anything useful. The "hover" when it is selected is part of the style; all the buttons work that way.
